package.json

```diff
@@ -23,12 +23,26 @@
       "no-undef": 0,
       "no-unused-vars": 0,
       "no-console": 0,
-      "indent": ["error", 2],
+      "indent": [
+        "error",
+        2
+      ],
```

Comment:
this and the next changes are whitespace-only changes
do we actually need that?

Reply:
We can think about. The good on this is that the linter could run some fixes automatically if configured.

Reply:
I was talking about the `package.json` file itself. The whitespace of that one changed without having other changes. do we actually need that?